data/json/bionics.json

```diff
@@ -678,17 +678,22 @@
       "style_aikido",
       "style_barbaran",
       "style_biojutsu",
+      "style_bojutsu",
       "style_boxing",
       "style_capoeira",
       "style_centipede",
       "style_crane",
       "style_dragon",
       "style_eskrima",
       "style_fencing",
+      "style_medievalpole",
       "style_judo",
       "style_karate",
       "style_krav_maga",
       "style_leopard",
+      "style_lizard",
+      "style_scorpion",
```

Comment:
```suggestion
```
These two were obsoleted from vanilla and moved to Mythical Martial Arts

Reply:
Both the style lizard and scorpion are listed in martialarts_fictional. Is there any reason why it wasn't a complete migration?

Reply:
Straight removal of them from vanilla files would break saves from people who had those arts pre-migration.

Reply:
Hmmm.

Digging though some of the files at all it doesn't seem like those fight styles are trying to be retired at all. They just aren't 'real' fighting styles  used in the real world. the books to learn them are still in the drop tables, and if they were being migrated, I would imagine they would at least be pulled into an obsolete json file.

Anyone else have an opinion on whether fictional fight styles should be included?

Reply:
You're duplicating the scorpion entry, see line 701. Also the martial art book for those style are not in any item groups, so they don't spawn and are as far as I can tell on their way out.